docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -0,0 +1,203 @@
+
+. Optional: Disable the system module in the {metricbeat}.
++
+--
+// tag::disable-system-module[]
+By default, the {metricbeat-ref}/metricbeat-module-system.html[system module] is
+enabled. The information it collects, however, is not shown on the
+*Stack Monitoring* page in {kib}. Unless you want to use that information for
+other purposes, run the following command:
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+metricbeat modules disable system
+----------------------------------------------------------------------
+// end::disable-system-module[] 
+--
+
+. Identify where to send the monitoring data. +
++
+--
+TIP: In production environments, we strongly recommend using a separate cluster 
+(referred to as the _monitoring cluster_) to store the data. Using a separate 
+monitoring cluster prevents production cluster outages from impacting your 
+ability to access your monitoring data. It also prevents monitoring activities 
+from impacting the performance of your production cluster.
+
+For example, specify the {es} output information in the {metricbeat} 
+configuration file (`metricbeat.yml`):
+
+[source,yaml]
+----------------------------------
+output.elasticsearch:
+  # Array of hosts to connect to.
+  hosts: ["http://es-mon-1:9200", "http://es-mon2:9200"] <1>
+  
+  # Optional protocol and basic auth credentials.
+  #protocol: "https"
+  #username: "elastic"
+  #password: "changeme"
+----------------------------------
+<1> In this example, the data is stored on a monitoring cluster with nodes 
+`es-mon-1` and `es-mon-2`.
+
+If you configured the monitoring cluster to use encrypted communications, you
+must access it via HTTPS. For example, use a `hosts` setting like
+`https://es-mon-1:9200`.
+
+IMPORTANT: The {es} {monitor-features} use ingest pipelines, therefore the
+cluster that stores the monitoring data must have at least one ingest node.
+
+If the {es} {security-features} are enabled on the monitoring cluster, you 
+must provide a valid user ID and password so that {metricbeat} can send metrics 
+successfully: 
+
+.. Create a user on the monitoring cluster that has the 
+`remote_monitoring_agent` {stack-ov}/built-in-roles.html[built-in role]. 
+Alternatively, use the `remote_monitoring_user` 
+{stack-ov}/built-in-users.html[built-in user]. 
++
+TIP: If you're using index lifecycle management, the remote monitoring user
+requires additional privileges to create and read indices. For more
+information, see `<<feature-roles>>`.
+
+.. Add the `username` and `password` settings to the {es} output information in 
+the {metricbeat} configuration file.
+
+For more information about these configuration options, see 
+{metricbeat-ref}/elasticsearch-output.html[Configure the {es} output].
+--
+
+. {metricbeat-ref}/metricbeat-starting.html[Start {metricbeat}] to begin
+collecting monitoring data. 
+
+. {kibana-ref}/monitoring-data.html[View the monitoring data in {kib}]. 
```

Comment:
I can't find the source of this link so I'm adding this comment here:

<img width="811" alt="confuse" src="https://user-images.githubusercontent.com/111616/66046510-70241e80-e515-11e9-91b0-60db866cac66.png">

Basically, the reason I find this confusing is that at this point, you're done and you should NOT be enabling internal collection.

IMHO, we need to either make this totally clear somehow or remove this sequence of links when clicking through.

Reply:
Excellent point! Currently, we have no control over the Previous and Next links. I discussed this with @nik9000, and he created an issue: https://github.com/elastic/docs/issues/1242